homeassistant/components/elmax/config_flow.py

```diff
@@ -170,82 +177,65 @@ async def async_step_panels(
 
     async def async_step_reauth(self, entry_data: Mapping[str, Any]) -> FlowResult:
         """Perform reauth upon an API authentication error."""
-        self._reauth_username = entry_data.get(CONF_ELMAX_USERNAME)
-        self._reauth_panelid = entry_data.get(CONF_ELMAX_PANEL_ID)
+        self._entry = self.hass.config_entries.async_get_entry(self.context["entry_id"])  # type: ignore[assignment]
         return await self.async_step_reauth_confirm()
 
-    async def async_step_reauth_confirm(self, user_input=None):
+    async def async_step_reauth_confirm(self, user_input=None) -> FlowResult:
```

Comment:
user_input is missing typing

Reply:
Thanks for the spot: I was actually missing to push some commits as the ae2653b9af63af23330c3ff4389b400c95cf58da which addresses specifically this issue. 

Should be fine now. Thanks!